browser/metrics/metrics_reporting_util.cc

```diff
@@ -26,10 +26,8 @@ bool GetDefaultPrefValueForMetricsReporting() {
       return true;
     case version_info::Channel::UNKNOWN:
       return false;
-    default:
-      NOTREACHED_IN_MIGRATION();
-      return false;
   }
+  NOTREACHED();
```

Comment:
nit: `NOTREACHED() << "Unexpected value for channel: "<< base::to_underlying(channel);`

Reply:
Done